lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -501,7 +501,11 @@ void ResourceMap::loadDefaultConfiguration()
   // Mesh parameters
   setAsScalar( "Mesh-VertexEpsilon", 1.0e-12 );
   setAsUnsignedInteger( "Mesh-LargeSize", 5000 );
-  setAsUnsignedInteger( "Mesh-UseKDTree", 1 );
+  setAsBool( "Mesh-UseKDTree", true );
```

Comment:
It must also be changed in resource file, or better yet, remove it, this entry is never used.

Reply:
@dbarbier It was there to prevent the mesh to build its kd-tree on a systematic basis, but forgot to propagate it in the code. Your solution is much better than this hack, so I will drop this entry.